http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/ProxyConfiguration.java

```diff
@@ -41,6 +41,7 @@ public final class ProxyConfiguration implements ToCopyableBuilder<ProxyConfigur
     private final String username;
     private final String password;
     private final Boolean useSystemPropertyValues;
+    private static final String HTTPS = "https";
```

Comment:
Can we use https://github.com/aws/aws-sdk-java-v2/blob/master/core/sdk-core/src/main/java/software/amazon/awssdk/core/Protocol.java#L29?  (Sorry, I should've mentioned it earlier)

Reply:
core not accessible to http client modules